dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/acl/DefaultTrackedEntityProgramOwnerService.java

```diff
@@ -117,4 +132,8 @@ private void updateTrackedEntityProgramOwner(
   public TrackedEntityProgramOwner getTrackedEntityProgramOwner(TrackedEntity te, Program program) {
     return trackedEntityProgramOwnerStore.getTrackedEntityProgramOwner(te, program);
   }
+
+  private String getOwnershipCacheKey(TrackedEntity trackedEntity, Program program) {
```

Comment:
It would be good to have this extracted somewhere and use the same code to create the key here and in `DefaultTrackerOwnershipManager`